.github/workflows/win-exe.yml

```diff
@@ -261,9 +261,14 @@ jobs:
         shell: pwsh
       - name: Download WinFsp
         run: |
-          $winfspUrl = (Select-String -Path ".\dist\win\bundle\resources\winFspMetaData.wxi" -Pattern '<\?define BundledWinFspDownloadLink="(.+)".*?>').Matches.Groups[1].Value
+          $winfspUrl = (Select-String -Path ".\dist\win\bundle\resources\winfsp.url" -Pattern 'URL=(https://.+)$').Matches.Groups[1].Value
```

Comment:
this is a rather unconventional approach. wouldn't it be way easier to just define constants for the URL at the top of the CI and/or build script?

Reply:
My approach was to have one place to store the URL. Otherwise, i need to remember two places when WinFsp is updated.

Reply:
Same argument would be valid for downloading the OpenJFX .jmod files, but still it is preferable to have a common approach for all distribution packages rather than having to remember a set of mixed approaches on Windows.

Cracking all nuts with the same stone is sometimes simpler than using a sophisticated automatic nut cracker that can only handle walnuts.